internal/provisioner/objects/dataplane/dataplane.go

```diff
@@ -212,7 +202,6 @@ func desiredContainers(contour *model.Contour, contourImage, envoyImage string)
 				SuccessThreshold:    int32(1),
 				TimeoutSeconds:      int32(1),
 			},
-			Ports: ports,
```

Comment:
In testing this, I realized that every time a new Gateway Listener with a new port is added, it triggers a rollout of Envoy because the container ports change. Here, I've stopped setting container ports on the Envoy workload entirely, since it's not required in order to expose the port anyway and mostly serves as documentation. Definitely interested in folks' thoughts here.

Reply:
@izturn @wilsonwu @Jean-Daniel I'd be interested in any input you have here. Let me know if you need more context!